src/renderer/components/App.tsx

```diff
@@ -50,11 +50,13 @@ export default class App extends React.Component<any, undefined> {
 
     // Called when files are droped on the dropzone
     public onDrop(acceptedFiles: File[]) {
+        const tabfile: File[] = this.cleanTabFile(acceptedFiles);
```

Comment:
Just out of interest, why the term "tab"? (sorry if I am missing something obvious ;)

Reply:
tabfile is for table of file. i can change this variable name if it isn't enough relevant.

Reply:
You mean an "array" / "list"?
In English, a "table" is a matrix (or something you have dinner at ;)

Reply:
2D matrix = rows+columns